src/domains/ecommerce-app/env/weu-uat/terraform.tfvars

```diff
@@ -37,3 +37,19 @@ ecommerce_xpay_psps_list = "CIPBITMM"
 ecommerce_vpos_psps_list = "testPSP3,testPSP4"
 
 dns_zone_checkout = "uat.checkout"
+
+cidr_subnet_ecommerce_functions = "10.1.155.0/24"
```

Comment:
ditto

Reply:
Fixed in https://github.com/pagopa/pagopa-infra/commit/bcf8f6ba978923b043875389e758d916d2db2d11